pkg/model/resources/nodeup.go

```diff
@@ -180,6 +180,43 @@ cat > kube_env.yaml << '__EOF_KUBE_ENV'
 __EOF_KUBE_ENV
 
 download-release
+
+until {{ KubeKeys }}; do
```

Comment:
nit: could this end up in a loop of death when s3 permissions on the instance don't match?

Reply:
yes, need break it

Reply:
`break` after 10 times try added into the loop